test/ufrag.c

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024 Paul-Louis Ageneau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#include "juice/juice.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+int test_ufrag() {
+	juice_agent_t *agent;
+	bool success = true;
+
+	juice_set_log_level(JUICE_LOG_LEVEL_DEBUG);
+
+	// Create agent
+	juice_config_t config;
+	memset(&config, 0, sizeof(config));
+
+	agent = juice_create(&config);
+
+	// Set local ICE attributes
+	juice_set_local_ice_attributes(agent, "ufrag", "pwd");
```

Comment:
Thank you for adding a test, but [it fails](https://github.com/paullouisageneau/libjuice/actions/runs/9051355044/job/24867788262?pr=243#step:6:913) because "pwd" is too short.

Reply:
sorry, I forgot it.